index.html

```diff
@@ -155,13 +155,31 @@
    .h1-section{
       text-decoration:underline;
        text-shadow:2px 1px yellow;
+       padding: 15px;
      } 
+     ol li{
+ list-style:none;
+ }
+#skillSec li{
+display:inline-block;
+width:200px;
+height:100px;
```

Comment:
This may be brittle -- If you have text that is longer than expected, what will happen?